docs/deployment-guide.md

```diff
@@ -464,6 +518,28 @@ all:
 All the `_environment` variables defined in roles are dictionaries, and all their keys are added to the relevant components start script thus allowing you to define any number of environment variables. Key values are a list of strings to allow for easier manipulation.
 When overriding the default env vars you should make sure you're not retiring important ones so always take a look at the ``roles/ROLE_NAME/defaults/main.yml` file first.
 
+### Single Sign On (Keycloak)
+
+> We are providing this integration as a preview and is currently not supported.
```

Comment:
Instead of talking of "support" we should instead state that we provide this role as-is for convenience only so people can experience the SSO  workflow and understand how they can configure their own keycloak if they want to.